src/components/import/partials/import.html

```diff
@@ -20,10 +20,11 @@
   <div ng-if="wmtsGetCap"
        ngeo-wmts-get-cap="wmtsGetCap"
        ngeo-wmts-get-cap-map="::map"
+       ngeo-wmts-get-cap-url="::getCapUrl"
```

Comment:
Are you sure of the `::`  ? if we change the url, the scope value must reinterpreted, no?